test/functional/core/job_spec.lua

```diff
@@ -15,14 +15,25 @@ local expect_twostreams = helpers.expect_twostreams
 local expect_msg_seq = helpers.expect_msg_seq
 local Screen = require('test.functional.ui.screen')
 
+-- Find process with given pid
+local function os_ps(pid)
+  return os.execute((iswin()
+    and 'tasklist /nh /fi "PID eq '..pid..'" | findstr '..pid..' > nul'
+    or  'ps -p '..pid..' > /dev/null'))
+end
```

Comment:
#8107 provides `nvim_get_proc({pid})` which can be used in the tests via `helpers.meths.get_proc()`.

Reply:
I need to return an exit code though.
Need something like `nvim_get_proc(pid) ~= nil and 1 or 0`.

Reply:
Why do you need an exit code? If `nvim_get_proc` returns `NIL` then the process wasn't found.